Models/book.rb

```diff
@@ -1,8 +1,15 @@
+# frozen_string_literal: true
+
+require_relative 'author'
+
 class Book
-  attr_accessor title, autor
+  attr_reader :title, :autor
 
   def initialize(title, author)
+    raise StandardError, 'Author is incorrect' unless author.is_a?(Author)
+    raise StandardError, 'Title is incorrect' unless title.is_a?(String) && title != ''
```

Comment:
move to private method or even in reusable service

Reply:
fixed